packages/components/entity-list/src/EntityListItem/EntityListItem.styles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { EntityListItemProps } from './EntityListItem';
+
+export const getEntityListItemStyles = () => ({
+  root: (props: Pick<EntityListItemProps, 'isDragActive'>) =>
+    css({
+      display: 'flex',
+      borderBottom: `1px solid ${tokens.gray200}`,
+      position: 'relative',
+      transition: `${tokens.transitionDurationShort} ${tokens.transitionEasingDefault}`,
+      transitionProperty: 'box-shadow, background-color',
+      backgroundColor: tokens.colorWhite,
+      ...(props.isDragActive
+        ? {
+            boxShadow: tokens.boxShadowHeavy,
+          }
+        : {}),
+
+      '&:hover': {
+        backgroundColor: tokens.gray100,
+      },
+    }),
+  card: css({
+    display: 'flex',
+    textDecoration: 'none',
+    width: '100%',
+    minHeight: '3.75rem',
+    padding: tokens.spacingXs,
+    border: 'none',
+    background: 'none',
+    textAlign: 'left',
+  }),
+  content: css({
+    flexGrow: 1,
+    minWidth: 0,
+  }),
+  media: css({
+    display: 'flex',
+    flexShrink: 0,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: tokens.gray200,
+    width: '2.875rem',
```

Comment:
Let's change the width and the height of the media here to `spacing-2xl`

That will result in the EntityList item with a computed height of `64px`, which is equal to our `spacing-3xl`.

Reply:
Can I ask what the reason for such changes? I don't think it's so necessary to couple everything with our spacing tokens, especially if it's about sizes not the spacing itself. So I wouldn't change the old design just for the sake of reusing spacing tokens. WDYT @domarku?

Reply:
As discussed in standup today, the reason to keep sizes coupled with our spacing tokens is to keep everything on a modular scale and have all of our elements better sit on a visual grid. I don't mind if the media height and width is a hardcoded value, but would still go for `3rem` (spacing-2xl).